### ISSUE TITLE
General discussion / to do list

### ISSUE THREAD

**Issue by erwinlambert** [AJueling/melt#3]
So the sheet model now works with the IdealGeometry as input.

An example for running and plotting the model is in Sheet_common_input.ipynb

To suppress printing of the output: 
sheet.diagint = 1e10 #standard: every 100 time steps

To do a quick run:
sheet.days = 0.1 #standard: 12 days, though full equilibration takes longer

These parameters can be updated after calling the SheetModel and before running .compute()

**Comment by erwinlambert** [comment]
Note: I fixed a bug in the sheet model (there was non-zero convergence across boundaries).

**Comment by AJueling** [comment]
The three models now work with the idealized input (you can look at the MIP.ipynb). However, the values and patterns don't all make sense (although they all have the right order of magnitude), so I will have to look for the cause of this.

**Comment by erwinlambert** [comment]
Hmm I do notice that there's some asymmetry in the code. test1 works fine, any rotation of the system goes wrong... So for now, I'd say focus on test1 (and test3)

**Comment by AJueling** [comment]
Yes, there was a problem in assigning the boxes in the right direction. I fixed that now and the results are now equal between `test1` and `test2`.

**Comment by erwinlambert** [comment]
OK, there's a similar problem in the sheet model (which I meant). Looks like test1 and test2 should be equal and test3 should work as well. Ice shelf front in the south or west poses an issue. Some bug in the masking with u/v at N/E points. Shouldn't take too long to fix, but won't get around to it today as I'm about to head to KNMI

**Comment by AJueling** [comment]
Meeting notes

- [x] (E) idealized temperature/salinity profiles with tanh with fixed surface temperature at freezing point and variable thermocline depth and deep temperature, and density profile (sets salinity) -> output xarray Dataset with z-dimension (coordinates negative to be consistent with draft)
- [x] (E/A) test sensitivity to density profile, ideally choose one interesting one
- [x] (E/A) two idealized geometries (test1+test3) + realistic
- [x] (E/A) plot avg. melt and maximum melt as a function of deep temperature and thermocline depth
- [x] models: 
   - (A) Simple ({linear}, {quadratic}, non-local quadratic), quasi-Levermann, Plume, PICO, PICOP
   - (E) Sheet
- [x] (A) Plume model depth of entrained properties: now draft, maybe draft+thickness
- [x] (E) upload Sheet model
- [x] (A) extend streamlines to areas without satellite data
- [x] (A) advection grounding line distance

presentation Monday
- [x] (E) send Sheet model results for MIP plot
- [x] (A) introduce model intercomparison project
- [x] (E) show Sheet model and present NEMO project ideas
- pre-meeting Monday 11:30

presentation Wednesday
- [x] (A/E) MIP paper outline
- [x] (E) present Sheet model
- [x] (E) new forcing profiles
- general discussion points: NEMO plans, PROTECT meeting

**Comment by AJueling** [comment]
Meeting 11.12.

Status:
- separated geometry from forcing from models -> makes extension easy
- all models work for now, not validated/calibrated well yet

To Do
- [x] (A) homogenize masks
- [ ] (A) check angle calculation
- [ ] (A) implement MISOMIP geometry (ocean1/2) + forcing (WARM+Favier), clean up old idealized geometries
- [x] (A) extend streamlines to no-radar areas
- [ ] (A/E) possibly coarsen large ice shelves
- [x] (A) finish grounding line distance algorithm
- [ ] (A) PICO thermal driving now @min(draft), can choose 500 or 700 m
- [ ] (E) stabilize Layer model (advection scheme, parameters)
- [ ] (E) separate plot functions, clean Layer code
- [x] (A) forcing: potential vs. in-situ temperature
- [x] (A/E) consistent names for output
    - output file name: {model}_{geometry}_{forcing}_{run}.nc
    - field names:
        - coordinates `x`, `y` [m]
        - `mask` (0,1,3)
        - melt pattern `melt` (x,y) [m/yr], average melt `mav` () [m/y]
        - `u`/`v` for Layer, Plume, PICOP

- [ ] (all) discuss model coherence ($\gamma_T$/M0, tides, entrainment)
- [ ] (all) updated Plume model from Adrian
- [ ] (all) validation with Paul's model output
- [x] (A) email to all: project update (MISOMIP geometry + forcing, Amundsen + Totten, realistic forcing: good profiles, decadal variability), invitation to paper, continue January, idealized experiments by end of January, (+knmi email address)

**Comment by erwinlambert** [comment]
General discussion points remaining, which are not allocated in separate issues:

(To be picked up in January):

- Thermal driving of PICO: min(draft), 500m, 700m?
- General forcing: potential vs in-situ temperature
- Discuss model coherence (gamma_T vs M0, tides, entrainment)
- Update plume model with local/non-local temperature forcing?
- Validation with Paul's model output
